testing/zsh-syntax-highlighting/APKBUILD

```diff
@@ -0,0 +1,28 @@
+# Contributor: Dawid Dziurla <dawidd0811@gmail.com>
+# Maintainer: Dawid Dziurla <dawidd0811@gmail.com>
+pkgname=zsh-syntax-highlighting
+pkgver=0.5.0
+pkgrel=0
+pkgdesc="Fish shell like syntax highlighting for Zsh."
+url="https://github.com/zsh-users/zsh-syntax-highlighting"
+arch="noarch"
+license="BSD-3"
+depends="zsh"
+makedepends=""
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::$url/archive/$pkgver.tar.gz"
+options="!check"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build()
+{
+	cd "$builddir"
+	make || return 1
```

Comment:
`|| return 1` is unnecessary, abuild now runs with `set -e`. Also it doesn’t make any sense after the last command of the function.